Makefile

```diff
@@ -55,3 +55,33 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+BENCH_PATH ?= bench
+TEST_PATH ?= ./...
+
+.PHONY: bench-deps
+bench-deps:
+	rm -rf ${BENCH_PATH}
+	mkdir ${BENCH_PATH}
+	go clean -testcache
+	cd ${BENCH_PATH} && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	rm -rf ${BENCH_PATH}/perf
+
+.PHONY: bench
+bench:
+	go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 \
+		| grep -v pkg: | grep -v ok | grep -v ? | tee ${BENCH_PATH}/bench.txt
+	benchstat ${BENCH_PATH}/bench.txt
+
+.PHONY: perf-reference
+perf-reference:
+	cd ${BENCH_PATH} && git clone https://github.com/tarantool/go-tarantool.git
+	cd ${BENCH_PATH}/go-tarantool && go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} \
+ 		-count=${COUNT} 2>&1 | grep -v pkg: | grep -v ok | grep -v ? | tee ../reference.txt
```

Comment:
`../reference.txt` - put it to a variable, it is used in `perf-diff` target too

Reply:
Done, `bench.txt` moved to variable too.